.github/workflows/ci.yaml

```diff
@@ -103,6 +121,8 @@ jobs:
         run: sudo apt-get install -y qemu-system-x86
       - if: matrix.arch == 'arm64'
         run: sudo apt-get install -y qemu-system-arm
+      - if: matrix.arch == 'ppc64le'
+        run: sudo apt-get install -y qemu-system-ppc64
```

Comment:
`qemu-system-ppc` seems to be the package name, but it seems to install the same set.

Reply:
`qemu-system-ppc64` is a virtual package, I figure it's better to explicitly ask for the 64-bit version we need.

Reply:
Don't worry, it is just a nit: I was using the "real" package name in the other two arches, so I mentioned it. I typically look up in the repo index online the package names, and it does not seem to show the virtual ones there.